leakcanary-android-core/src/main/java/leakcanary/internal/InternalLeakCanary.kt

```diff
@@ -76,6 +83,33 @@ internal object InternalLeakCanary : (Application) -> Unit, OnObjectRetainedList
         onHeapAnalyzedListener = OnHeapAnalyzedListener {}
     )
 
+  internal enum class FormFactor {
+    MOBILE,
```

Comment:
So tablets are MOBILE :) ?

Reply:
They are, but I share your confusion here! I think we need a better name that will cover both mobile and tablet. Is there a commonly accepted term to describe these two?